.ci.yaml

```diff
@@ -90,6 +90,14 @@ platform_properties:
       device_type: none
       cpu: arm64
       xcode: 14a5294e # xcode 14.0 beta 5
+  mac_benchmark:
```

Comment:
evidently a `dependencies` key is required:
```
Traceback (most recent call last):
  //main.star:249:21: in <toplevel>
  //flutter_config.star:59:21: in _setup
  //lib/ci_yaml/ci_yaml.star:471:14: in _generate
  //lib/ci_yaml/ci_yaml.star:371:40: in _builders
  //lib/ci_yaml/ci_yaml.star:227:34: in _properties
Error: key "dependencies" not in dict
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8790222719971180513/+/u/luci_generate/stdout